{{cookiecutter.project_slug}}/clients/vue3/src/views/PageNotFound.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="page-not-found">
+    <h1 class="page-not-found__error">404</h1>
+    <img alt="Project Logo" src="../assets/logo.png" />
+    <h2 class="page-not-found__header">Oops!<br />We couldn't find the page you're looking for.</h2>
+    <p class="page-not-found__route">
+      Click <router-link to="/">here</router-link> to return to the home page.
+    </p>
```

Comment:
**Info:** Just went with some basic text to indicate that the page the user has landed on does not exist (as well as giving a clickable/focusable redirect link for users), but this can be updated to something else should additional/different verbiage be preferred.